admin-dev/themes/default/js/bundle/product/form.js

```diff
@@ -838,6 +838,9 @@ var form = (function() {
       });
 
       elem.submit(function(event) {
+        $.each($('input.attribute_wholesale_price, input.attribute_priceTE, input.attribute_priceTI, input.attribute_unity, input.attribute_weight'), function () {
+          $(this).val($(this).val().replace("−", "-"));
+        });
```

Comment:
What do you think about extracting that part into a function that you could call from there and from line `860`? It would be more DRY :)

Reply:
Good idea.